### PR TITLE
Update DDO Menu of Tools to account for COVID-19 Changes

### PR DESCRIPTION
**Pull request by sraby**
This PR updates our DDO Menu of Tools to accommodate the many changes brought by the COVID-19 outbreak in NYC. Specifically:

1. I updated the prioritization of our DDO cards based on team feedback
2. I removed all "recommendations" so all cards now appear in a fixed order under a single header called "Actions"
3. I implemented temporary content changes for HP, LOC, and EFNYC cards.

### CODE REVIEW

**Comment by sraby** [comment]
@toolness this is ready to go for when we need it. Feel free to review if you have a free moment and are interested.

File: frontend/sass/_data-driven-onboarding.scss [hunk 1]
```diff
@@ -58,7 +58,8 @@
 
 .jf-ddo-results {
     .jf-ddo-card {
-        p.subtitle {
+        p.subtitle, span.subtitle {
```
Comment (sraby) [inline frontend/sass/_data-driven-onboarding.scss]: Here, I needed some way to mimic the styling of the paragraphs of text within a DDO card without actually using `<p>` tags. It seems like by default, all blocks of texts in the "Indicators" array get wrapped in a `<p>` tag and then each block of "fallback text" also get's wrapped in one. I wanted to break up the text within both of these and thought it was overkill to refactor things so I added this styling to `<span>` to mimic the spacing of the paragraph. Since `<p>`'s are `display: block` by default, this shouldn't have any effect on pre-existing styling.

Comment (toolness) [reply frontend/sass/_data-driven-onboarding.scss]: Ah yeah, that is an annoying thing I ran into earlier too.  This seems like a coolio fix tho and I don't _think_ it should affect accessibility!

File: frontend/lib/pages/data-driven-onboarding.tsx [hunk 2]
```diff
@@ -23,12 +23,14 @@ const PLACEHOLDER_IMG = 'frontend/img/96x96.png';
...
+const VIOLATIONS_HIGH_PRIORITY = 50;
 const COMPLAINTS_PRIORITY = 40;
-const VIOLATIONS_PRIORITY = 30;
-const RENT_HISTORY_PRIORITY = 20;
-const WOW_PRIORITY = 10;
+const EFNYC_PRIORITY = 30;
+const WOW_PRIORITY = 20;
+const RENT_HISTORY_PRIORITY = 10;
+
+const DISABLE_RECOMMENDATIONS = true;
```
Comment (toolness) [inline frontend/lib/pages/data-driven-onboarding.tsx]: Maybe we should add a comment here just so future folks know why it's being put in there?  Eh, not a big deal either way.

File: frontend/lib/pages/data-driven-onboarding.tsx [hunk 3]
```diff
@@ -266,6 +268,13 @@ const ACTION_CARDS: ActionCardPropsCreator[] = [
     };
   },
   function hpAction(data): ActionCardProps {
+    // Default content temporarily implemented during COVID-19 Outbreak
+    const covidMessage = <>
+        <span className="subtitle">Due to the Covid-19 health crisis, Housing Courts in New York City are closed. 
+          You can still make the forms to take your landlord to court but you will not be able to file them until the courts re-open.</span>
+        <span className="subtitle">If you are facing an emergency such as lack of heat and/or hot water, call the <strong>Housing Court Answers Hotline at (212) 962-4795</strong> to get assistance Mon-Fri, 9am-5pm.
```
Comment (toolness) [inline frontend/lib/pages/data-driven-onboarding.tsx]: Huh, I'm curious if we want to wrap that phone number in a [`tel:` link](https://css-tricks.com/the-current-state-of-telephone-links/).  That way if a mobile user clicks on it, it should start their phone app, so they don't have to write down the number and then visit their phone app and manually enter it.

File: frontend/lib/pages/data-driven-onboarding.tsx [hunk 4]
```diff
@@ -317,21 +320,24 @@ const ACTION_CARDS: ActionCardPropsCreator[] = [
     };
   },
   function evictionFreeNyc(data): ActionCardProps {
+    // Default content temporarily implemented during COVID-19 Outbreak
+    const covidMessage = <>
+      An Eviction Moratorium is in place in NY State due to the Covid-19 public health crisis. 
+      All courts that hear eviction cases are closed. This means you <b>cannot be evicted for any reason</b>. 
+    </>
+    const covidCtaText = "Learn more";
```
Comment (toolness) [inline frontend/lib/pages/data-driven-onboarding.tsx]: Just FYI, "Learn more" is one of those not-great-for-accessibility link texts, because assistive tech (AT) users who are skimming a page by going through each link's text are just going to hear "learn more" without any context...  It's not a dealbreaker though and there are ways for us to append screenreader-only text to it so that for sighted users it says "Learn more" but for AT users it will say "Learn more about the eviction moratorium"... but because of the abstraction layer we've put in here, that won't be super straightforward.

So in other words, if it's easy to change "Learn more" to something a bit more accessible that also wouldn't be weird to sighted users, that'd be ideal, but otherwise I wouldn't worry about it right now (though it would be good to file an issue so we can address it later... same w/ the "Learn more" on the banner from #1047 for that matter, I overlooked it when I reviewed that one!).

File: frontend/lib/pages/data-driven-onboarding.tsx [hunk 5]
```diff
@@ -342,9 +348,13 @@ function compareActionCardProps(a: ActionCardProps, b: ActionCardProps): number
...
 
 function getSortedActionCards(data: DDOData): { recommended: ActionCardProps[], other: ActionCardProps[] } {
-  const actionCardProps = ACTION_CARDS.map(propsCreator => propsCreator(data)).map(props => (
-    props.imageStaticURL ? props : {...props, imageStaticURL: SHOW_PLACEHOLDER_IMG ? PLACEHOLDER_IMG : undefined}
-  ));
+  const actionCardProps = ACTION_CARDS
+    .map(propsCreator => propsCreator(data))
+      .map(props => {
+        if (DISABLE_RECOMMENDATIONS) { props.isRecommended = false; }
+        return (props.imageStaticURL ? props : {...props, imageStaticURL: SHOW_PLACEHOLDER_IMG ? PLACEHOLDER_IMG : undefined});
```
Comment (toolness) [inline frontend/lib/pages/data-driven-onboarding.tsx]: Hmm, this feels a bit hard to read but I'm not sure of any particularly good ways to make it more readable.  One way might be to convert it all to something that uses spread notation like:

```js
return {
  ...props,
  isRecommended: DISABLE_RECOMMENDATIONS ? false : props.isRecommended,
  imageStaticURL: props.imageStaticURL || (SHOW_PLACEHOLDER_IMG ? PLACEHOLDER_IMG : undefined),
};
```

Uhh I'm not even sure if that's logically equivalent though, my brain hurts now.  Don't worry about it I guess.

File: frontend/lib/pages/data-driven-onboarding.tsx (hunk 4 above)
Comment (sraby) [reply frontend/lib/pages/data-driven-onboarding.tsx]: Marking this as an issue to overhaul later! https://github.com/JustFixNYC/tenants2/issues/1067

File: frontend/sass/_data-driven-onboarding.scss (hunk 1 above)
Comment (sraby) [reply frontend/sass/_data-driven-onboarding.scss]: great!

File: frontend/lib/pages/data-driven-onboarding.tsx (hunk 2 above)
Comment (sraby) [reply frontend/lib/pages/data-driven-onboarding.tsx]: 👍

File: frontend/lib/pages/data-driven-onboarding.tsx (hunk 3 above)
Comment (sraby) [reply frontend/lib/pages/data-driven-onboarding.tsx]: good idea!
